### PR TITLE
Style images

### PR DESCRIPTION
**Pull request by Artemis21**
Add styling for images to each of the themes.

### CODE REVIEW

File: jolly-jellyfish/src/static/filters.svg [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+-->
+
+<svg xmlns="http://www.w3.org/2000/svg" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" width="0" height="0">
+    <filter id="red">
+        <feColorMatrix type="matrix" values="
+            1   0.6 0.6 0 1
+            0.3 0   0   0 0
+            0.3 0   0   0 0
+            0   0   0   1 0
+        "/>
```
Comment (A5rocks) [inline jolly-jellyfish/src/static/filters.svg]: could you add a comment about what these values mean?

File: jolly-jellyfish/src/static/filters.svg [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    </filter>
+    <filter id="yellow">
+        <feColorMatrix type="matrix" values="
+            0.5  0.5 0.3 0 0.5
+            0.5  0.5 0.3 0 0.5
+            0.15 0   0   0 0
+            0    0   0   1 0
+        "/>
```
Comment (A5rocks) [inline jolly-jellyfish/src/static/filters.svg]: see above

File: jolly-jellyfish/src/static/themes/console.css [hunk 3]
```diff
@@ -13,6 +13,43 @@ body {
...
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  z-index: 5;
+  background: #000d;
+}
+
+.img-wrapper:hover::before {
+  content: "<";
```
Comment (A5rocks) [inline jolly-jellyfish/src/static/themes/console.css:52]: not sure if I like this... maybe we could adjust the transparency (using some css transitions)? (of course, if it looks great ignore this)

File: jolly-jellyfish/src/static/themes/unicorn.css [hunk 4]
```diff
@@ -7,17 +7,18 @@
...
+    url("../images/unicorn1.gif") left center / 33% 33% no-repeat,
+    url("../images/unicorn2.gif") left bottom / 33% 33% no-repeat,
+    url("../images/unicorn3.gif") center top / 33% 33% no-repeat,
+    url("../images/unicorn4.gif") center center / 33% 33% no-repeat,
+    url("../images/unicorn5.gif") center bottom / 33% 33% no-repeat,
+    url("../images/unicorn6.gif") right top / 33% 33% no-repeat,
+    url("../images/unicorn7.gif") right center / 33% 33% no-repeat,
+    url("../images/unicorn8.gif") right bottom / 33% 33% no-repeat,
+    url("../images/unicorn9.gif") left top / 33% 33% no-repeat,
+    linear-gradient(to right, red, orange, yellow, green, blue, indigo, violet) 100% 100% repeat;
```
Comment (A5rocks) [inline jolly-jellyfish/src/static/themes/unicorn.css]: Not sure how much I like the small amount of indentation here... Maybe indent more? (there was another spot with this problem too... but I already passed it xd)

File: jolly-jellyfish/src/static/themes/win95.css [hunk 5]
```diff
@@ -3,6 +3,7 @@
  * A static file "/static/fonts/hellovetica.woff2" should exist,
  * but it will fall back on "News Cycle" from Google Fonts if not.
  * Static files "/static/images/win95-toolbar.png" and "/static/images/win95-close.png" should also exist.
+ * Requires that images be wrapped in divs.
```
Comment (A5rocks) [inline jolly-jellyfish/src/static/themes/win95.css:6]: Mind writing what class is required? (if one is... I didn't see any new things about divs being related to images in the file...)

File: jolly-jellyfish/src/static/themes/win95.css [hunk 6]
```diff
@@ -28,6 +29,20 @@ body {
   margin: 30px auto 60px auto;
 }
 
+img {
+  width: 100%;
+  height: 100px;
```
Comment (A5rocks) [inline jolly-jellyfish/src/static/themes/win95.css]: Should we really be using pixel measurements for something?

I'm not sure if it appears anywhere else though, so maybe we already are and I missed it...

**Review by A5rocks — COMMENTED** [review]
Can't comment on the design much, just a couple nitpicks :)

File: jolly-jellyfish/src/static/themes/console.css (hunk 3 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/console.css:52]: That exists as a button to close the modal

File: jolly-jellyfish/src/static/themes/unicorn.css (hunk 4 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/unicorn.css]: I will go back to the visual indent, sure

File: jolly-jellyfish/src/static/themes/console.css (hunk 3 above)
Comment (A5rocks) [reply jolly-jellyfish/src/static/themes/console.css:52]: Oh okay, nice.

File: jolly-jellyfish/src/static/themes/win95.css (hunk 5 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/win95.css:6]: This theme isn't picky about the class, but I do need to put that in the Console and Galaxy themes.

This theme does still require the divs though, to make 100% relative to the size of the image.

File: jolly-jellyfish/src/static/themes/win95.css (hunk 6 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/themes/win95.css]: There are quite a few things using pixels, I take your point for this one though, maybe I should change it to 2em or something.

File: jolly-jellyfish/src/static/filters.svg (hunk 1 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/filters.svg]: Added

File: jolly-jellyfish/src/static/filters.svg (hunk 2 above)
Comment (Artemis21) [reply jolly-jellyfish/src/static/filters.svg]: see above

**Review by A5rocks — APPROVED** [review]
LGTM from a cursory overview

**Pull request merged**